### PR TITLE
[143] Develop a Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by Lillian-1**
snap3

### CODE REVIEW

**Review by Ireti203 — CHANGES_REQUESTED** [review]
Why do you have two form.html files? You have one in the html-forms folder and another in the html folder. Kindly confirm there is no mix-up.

**Comment by Lillian-1** [comment]
> Why do you have two form.html files? You have one in the html-forms folder and another in the html folder. Kindly confirm there is no mix-up.

It's for linking the contact page and products page.

File: submissions/lillianNwaoha/html-forms/html/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        <img src="./images/img1.jfif" alt="born this way">
+        <h4>PRO MATTE FOUNDATION <br>N6,000</h4>
+        
+        <p>DESCRIPTION: With a soft and suede-like finish, Born this way foundation will have you covered and shine free all day. Creamy,liquid formula provides smooth and buildable coverage for both day and night time wear.</p>
+        <button>ADD TO CART </button>
+        <br>
+<br>
+        <img src="./images/img2.jfif" alt="fit me">
+        <h4>PRO MATTE FOUNDATION <br>N3,000</h4>
+        <p>DESCRIPTION: With a soft and suede-like finish, Born this way foundation will have you covered and shine free all day. Creamy,liquid formula provides smooth and buildable coverage for both day and night time wear.</p>
```
[the remark this answers is not in the document]
Comment (Lillian-1) [reply submissions/lillianNwaoha/html-forms/html/index.html:23]: i have corrected the errors.

**Comment by Lillian-1** [comment]
I have corrected it .


Sent from Yahoo Mail for iPhone


On Friday, November 27, 2020, 10:03 AM, Ireti203 <notifications@github.com> wrote:




@Ireti203 requested changes on this pull request.

Why do you have two form.html files? You have one in the html-forms folder and another in the html folder. Kindly confirm there is no mix-up.

In submissions/lillianNwaoha/html-forms/html/index.html:
> +<html>
+    <head>
+        <title>Lily's Place</title>
+    </head>
+    <body>
+        <h1> LILY'S MAKEUP STORE </h1>
+        <a href="home">Home</a>
+        <a href="product">Products</a>
+        <a href="/form.html">Contact Us</a> 
+
+        <h2>HOME TO THE BEST MAKEUP PRODUCTS</h2>
+        <h3>FOUNDATION PRODUCTS</h3>
+        born this way
+        <h4>PRO MATTE FOUNDATION <br>N6,000</h4>
+        
+        <p>DESCRIPTION: With a soft and suede-like finish, Born this way foundation will have you covered and shine free all day. Creamy,liquid formula provides smooth and buildable coverage for both day and night time wear.</p>

This line of code is too long. Break it to the next line

In submissions/lillianNwaoha/html-forms/html/index.html:
> +        <a href="home">Home</a>
+        <a href="product">Products</a>
+        <a href="/form.html">Contact Us</a> 
+
+        <h2>HOME TO THE BEST MAKEUP PRODUCTS</h2>
+        <h3>FOUNDATION PRODUCTS</h3>
+        born this way
+        <h4>PRO MATTE FOUNDATION <br>N6,000</h4>
+        
+        <p>DESCRIPTION: With a soft and suede-like finish, Born this way foundation will have you covered and shine free all day. Creamy,liquid formula provides smooth and buildable coverage for both day and night time wear.</p>
+        <button>ADD TO CART </button>
+        <br>
+<br>
+        fit me
+        <h4>PRO MATTE FOUNDATION <br>N3,000</h4>
+        <p>DESCRIPTION: With a soft and suede-like finish, Born this way foundation will have you covered and shine free all day. Creamy,liquid formula provides smooth and buildable coverage for both day and night time wear.</p>

same here

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**